configs/src/deployments/json/bscMainnet.json

```diff
@@ -0,0 +1,184 @@
+{
+  "v1.3.0": {
+    "AddresslistVotingSetup": {
+      "address": "0x6E924eA5864044D8642385683fFA5AD42FB687f2",
+      "blockNumber": 47580088,
+      "deploymentTx": "0x13c72f78734116883bca593df0ff20686e4de8a7fd92717dab28a9bd55500110"
+    },
+    "AddresslistVotingSetupImplementation": {
+      "address": "0xB287fA676198938fC2a098a546bF9e0368FF58Ad",
+      "blockNumber": 47580088,
+      "deploymentTx": "0x13c72f78734116883bca593df0ff20686e4de8a7fd92717dab28a9bd55500110"
+    },
+    "AdminSetup": {
+      "address": "0xB2A2b32b9d885C85d5b229C0509341c37CaE7483",
+      "blockNumber": 47580097,
+      "deploymentTx": "0x88107e86bf269973b392a72bbb5c753df5f0226e2d75990366ad11d67822ad7d"
+    },
+    "AdminSetupImplementation": {
+      "address": "0xfD3c7c2B4E71fCcce7587a9e1C93C706759e689f",
+      "blockNumber": 47580097,
+      "deploymentTx": "0x88107e86bf269973b392a72bbb5c753df5f0226e2d75990366ad11d67822ad7d"
+    },
+    "ManagementDAOProxy": {
+      "address": "0xF3AaA3372EbBf01b923a4Cc98Cd847126b3D73cA",
```

Comment:
Make sure that there is a multisig installed on it and it has the correct managers